packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -937,9 +963,27 @@ export const components: ThemeOptions["components"] = {
   MuiSelect: {
     defaultProps: {
       variant: "standard",
+      IconComponent: ChevronDownIcon,
     },
     styleOverrides: {
+      select: ({ theme }) => ({
+        paddingBlock: `calc(${theme.spacing(3)} - ${theme.mixins.borderWidth})`,
+        paddingInline: theme.spacing(3),
+
+        "&:focus": {
+          backgroundColor: "transparent",
+        },
+
+        ".MuiBox-root": {
+          display: "flex",
+          flexWrap: "wrap",
+          gap: theme.spacing(1),
+          marginBlock: `-${theme.spacing(2)}`,
```

Comment:
These negative margins account for the additional padding present when using `Tag` within multiselect.

Ideally, we would instead reduce the padding on the Select element when multiple values are present, but MUI doesn't provide a class to target.